translations/nl-be.json

```diff
@@ -526,5 +529,15 @@
   "search-legend-not-explanation": "NIET: Zoek op 'Antwerpen' en neem alle resultaten waar ook 'verkeer' in zit, weg",
   "search-legend-order-explanation": "Zoekt op de gehele reeks van woorden in die volgorde",
   "search-legend-wildcard-explanation": "Zoeken waarbij het opgegeven lemma de prefix is voor een woord",
-  "submitted-by": "- Indiener"
+  "submitted-by": "- Indiener",
+  "no-themes-selected-message": "Er zijn nog geen thema's geselecteerd. Bent u zeker dat u het kort bestek wilt opslaan?",
+  "manage-users-name": "Naam",
+  "manage-users-email": "Email",
+  "manage-users-telephone": "Telefoonnummer",
+  "manage-users-group": "Groep",
+  "manage-users-void": "VO-ID",
+  "manage-users-ovo": "OVO-code",
+  "manage-users-general-info": "Algemene informatie",
+  "manage-users-technical-info": "Technische informatie",
+  "manage-user": "Gebruiker: {firstName} {lastName}"
```

Comment:
Why add email, ovo-code and phone twice ?
The correct spelling is e-mail.

Reply:
Some translations already exists, could optionally be reused
"group": "Groep"
"national-number": "VO-ID",